next.json

```diff
@@ -1050,6 +1050,8 @@
                 "name": "configure",
                 "args": [
                     {"name": "format", "type": "texture format"},
+                    {"name": "allowedUsage", "type": "texture usage bit"},
+                    {"name": "initialUsage", "type": "texture usage bit"},
```

Comment:
We don't force an explicit transition, but we ended up making the transition implicit in `BeginRenderSubpass`. So we don't need `SwapChainBuilder::SetInitialUsage` for renderpass attachments, but it could be convenient for blits? Not a big deal either way.